bot.py

```diff
@@ -75,12 +75,11 @@ def choose_streams(self):
 		if len(live_streams) > 3:
 			chosen = []
 			for z in range(0, 3):
-				i = random.randint(0, len(live_streams))
```

Comment:
I think it would be better to use `i` everywhere instead of `i-1`. So it would be better to choose `i` like this `i = random.randint(0, len(live_streams) - 1)`